docs/core/tutorials/consuming-library-with-visual-studio.md

```diff
@@ -1,48 +1,47 @@
 ---
-title: Consume a .NET Standard library in Visual Studio 2017
-description: Build a .NET Core application that calls members of another class library with Visual Studio 2017.
+title: Consume a .NET Standard library in Visual Studio
+description: Build a .NET Core application that calls members of another class library with Visual Studio 2019.
 author: BillWagner
 ms.author: wiwagn
 ms.date: 06/05/2018
 ms.custom: "vs-dotnet, seodec18"
 ---
-# Consume a .NET Standard library in Visual Studio 2017
+# Consume a .NET Standard library in Visual Studio
 
-Once you've created a .NET Standard class library by following the steps in [Create a class library with .NET Standard in Visual Studio](./library-with-visual-studio.md), tested it in [Test a class library with .NET Core in Visual Studio](testing-library-with-visual-studio.md), and built a Release version of the library, the next step is to make it available to callers. You can do this in two ways:
+Once you've created a .NET Standard class library, tested it, and built a release version of the library, the next step is to make it available to callers. You can do this in two ways:
 
-* If the library will be used by a single solution (for example, if it's a component in a single large application), you can include it as a project in your solution.
-
-* If the library will be generally accessible, you can distribute it as a NuGet package.
+- If the library will be used by a single solution (for example, if it's a component in a single large application), you can include it as a project in your solution.
+- If the library will be publicly available, you can distribute it as a NuGet package.
 
 ## Include a library as a project in a solution
 
-Just as you included unit tests in the same solution as your class library, you can include your application as part of that solution. For example, you can use your class library in a console application that prompts the user to enter a string and reports whether its first character is uppercase:
+Just as you included unit tests in the same solution as your class library in [Test a .NET Standard library with .NET Core in Visual Studio 2017](testing-library-with-visual-studio.md), you can include your application as part of that solution. For example, you can use your class library in a console application that prompts the user to enter a string and reports whether its first character is uppercase:
```

Comment:
Can we remove 2017 as VS version from the link text?

Reply:
yes. don't review this yet. I'm far from being done. Just wanted to get some builds started.